### ISSUE TITLE
Are the experiment results in table2 and table3 performed on unlabeled data of the training set？ I found that k_means.py in your code to do just that?

### ISSUE THREAD

**Issue by ryylcc** [sgvaze/generalized-category-discovery#18]
Are the experiment results in table2 and table3 performed on unlabeled data of the training set？ I found that k_means.py in your code to do just that?

**Comment by xwen99** [comment]
> Are the experiment results in table2 and table3 performed on unlabeled data of the training set？ I found that k_means.py in your code to do just that?

@ryylcc yes, it is, this paper follows a transductive setting, the best model is picked on a hold-out validation set (called 'test' in the code), then evaluated on a test set (also the unlabeled part of the training set).

**Comment by sgvaze** [comment]
Hello, apologies for the late reply and thanks @xwen99 for the contribution. 

That is correct, we evaluate on the unlabelled training data. We think this aligns with the motivation of GCD 'given a dataset, some of which is labeled, categorize all unlabelled examples'
